subsystems/sidecar/index.js

```diff
@@ -443,6 +443,7 @@ class Sidecar extends ReadyResource {
     }
     if (typeof pkg?.pear?.links === 'object' && pkg.pear.links !== null) {
       for (const link of Object.values(pkg.pear.links)) {
+        if (link.slice(0, 7).toLowerCase() !== 'pear://') continue
```

Comment:
use parseLink to get proto

Reply:
@davidmarkclements Hmm.. parseLink does not seem to return proto. Did you mean `new URL()`?

Reply:
nice solution with key check, parseLink coming from entrypoints contains proto, but this key check will work either way - nice resolution thank you